charts/ccd-data-store-api/values.yaml

```diff
@@ -36,9 +36,12 @@ java:
 
     DEFINITION_CACHE_MAX_IDLE_SEC: 259200
     DEFINITION_CACHE_LATEST_VERSION_TTL_SEC: 1
+    DEFINITION_CACHE_JURISDICTION_TTL_SEC: 120
+    DEFINITION_CACHE_USER_TTL_SEC: 1800
+    DEFINITION_CACHE_CASE_DETAILS_TTL_SEC: 1800
+    DEFINITION_CACHE_DRAFT_TTL_SEC: 1800
```

Comment:
where do these come from?

Reply:
> where do these come from?

These are all newly introduced caching variables to promote more flexibility in our caching configuration.

Reply:
Removed all the newly introduced caching variables.